### PR TITLE
{phys}[system/system] AEDT v2024R1

### PR DESCRIPTION
**Pull request by appolloford**
(created using `eb --new-pr`)

Depends on:
* [x] https://github.com/easybuilders/easybuild-easyblocks/pull/3281

### CODE REVIEW

**Comment by appolloford** [comment]
Test report by @appolloford
Using easyblocks from PR(s) https://github.com/easybuilders/easybuild-easyblocks/pull/3281
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
vera40-1 - Linux Rocky Linux 8.9, x86_64, Intel(R) Xeon(R) Gold 6130 CPU @ 2.10GHz, 1 x NVIDIA Tesla T4, 550.54.14, Python 3.6.8
See https://gist.github.com/appolloford/347b4af792f14684f6b9cb30fd3ebb84 for a full test report.

File: easybuild/easyconfigs/a/AEDT/AEDT-2024R1.eb [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+Extractor, Ansys SIwave, and Ansys Icepak using electrical CAD (ECAD) and mechanical
+CAD (MCAD) workflows."""
+
+toolchain = SYSTEM
+
+sources = ['ELECTRONICS_%(version)s_LINX64.tgz']
+checksums = ['7b131adf981ebca1e2f4fe8e607e50323167b69e77180a0ab61b2759d57abca5']
+
+postinstallcmds = [
+    "echo '' > %s/VerifyOS.bash" % local_bin_dir,  # disable OS check
```
Comment (lexming) [inline easybuild/easyconfigs/a/AEDT/AEDT-2024R1.eb:19]: I suggest to move these to the easyblock given that it is static code not depending the contents of the easyconfig.

**Comment by appolloford** [comment]
Test report by @appolloford
Using easyblocks from PR(s) https://github.com/easybuilders/easybuild-easyblocks/pull/3281
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
vera-skylake-build - Linux Rocky Linux 8.9, x86_64, Intel Xeon Processor (Skylake, IBRS, no TSX), Python 3.6.8
See https://gist.github.com/appolloford/0018ade9ade81ba7b194b01cdfebb245 for a full test report.

**Review by Micket — APPROVED** [review]
lgtm

**Pull request merged**